CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
 The Google Pipelines API (PAPI) version 1 and 2 backends now offer the option of call cache blacklisting on a per-bucket basis.
 More info [here](http://cromwell.readthedocs.io/en/develop/CallCaching/#call-cache-copy-authorization-failure-prefix-blacklisting).
 
+### CWL
+
+Support `InputResourceRequirement` hint
```

Comment:
Was this the only code change required to make them work? TBH I just scanned down past all the CWL since I assume it's brought in from elsewhere?

Reply:
> Was this the only code change required to make them work?
Yes, there was a first batch of changes but already merged in a previous PR
>  I assume it's brought in from elsewhere?
Yes (I updated the description)